web/sites/default/config/field.field.node.contact.field_display_311.yml

```diff
@@ -0,0 +1,29 @@
+uuid: 050780e9-ed77-4ef4-b8bb-8d3d9952176b
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_display_311
+    - node.type.contact
+  module:
+    - custom_add_another
+third_party_settings:
+  custom_add_another:
+    custom_add_another: ''
+    custom_remove: ''
+id: node.contact.field_display_311
+field_name: field_display_311
+entity_type: node
+bundle: contact
+label: 'Display 311'
+description: 'Mark as True to display 311 info.'
```

Comment:
Added revised help text and permission check to the Jira issue.

Reply:
Fixed in new commit.